library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -54,9 +56,9 @@ internal class TransactionPayloadFragment :
     }
 
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
+            inflater: LayoutInflater,
+            container: ViewGroup?,
+            savedInstanceState: Bundle?
```

Comment:
Format is no need and not good

Reply:
> Format is no need and not good

Why do you leave such comments? We have ktLint for such issues. 
Please, stop spamming.